packages/sandbox/src/file_watching_sandbox.ts

```diff
@@ -88,15 +88,12 @@ export class FileWatchingSandbox extends EventEmitter implements Sandbox {
    */
   start = async (options: SandboxOptions) => {
     const watchDir = options.dir ?? './amplify';
-    options.dir && path.isAbsolute(options.dir)
-      ? options.dir
-      : path.join(process.cwd(), options.dir ?? './amplify');
     if (!fs.existsSync(watchDir)) {
-      this.printer.log(
-        `${watchDir} does not exist. please try running this command from your project root`,
-        LogLevel.ERROR
-      );
-      return;
+      throw new AmplifyUserError('DirectoryToWatchNotFoundError', {
```

Comment:
Not a blocker, but I think a slightly more generic name like `PathNotFoundError` would be better.

Reply:
Thanks, I rename it.